src/extensions/lens-main-extension.ts

```diff
@@ -10,10 +10,15 @@ import type { CatalogEntity } from "../common/catalog";
 import type { IObservableArray } from "mobx";
 import type { MenuRegistration } from "../main/menu/menu-registration";
 import type { TrayMenuRegistration } from "../main/tray/tray-menu-registration";
+import type { ShellEnvModifier } from "../main/shell-session/shell-env-modifier/shell-env-modifier-registration";
+import { noop } from "../common/utils";
+
 export class LensMainExtension extends LensExtension {
   appMenus: MenuRegistration[] = [];
   trayMenus: TrayMenuRegistration[] = [];
 
+  shellEnvModifier: ShellEnvModifier = noop;
```

Comment:
I don't think we want `noop` here, it would be better to have this be an optional field.